Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -224,6 +224,13 @@ enum class WhiteSpace {
     PreWrap,
 };
 
+enum class PointerEvents {
+    Auto,
+    None,
+    Initial,
+    Inherit
+};
```

Comment:
`initial` and `inherit` are special keywords, which don't need to be mentioned here. They're automatically replaced by actual values (`auto` or `none`) during the cascade, which is when values get assigned.

Also, please add these to [Properties.json](https://github.com/serenityos/serenity/blob/master/Userland/Libraries/LibWeb/CSS/Properties.json#L1074), like so:

```json
  "pointer-events": {
    "inherited": true,
    "initial": "auto",
    "valid-identifiers": [
      "auto",
      "none"
    ]
  },
```

This tells the CSS Parser that these values are allowed for the `pointer-events` property.